media-libs/ptex/ptex-2.1.28.ebuild

```diff
@@ -15,7 +15,7 @@ RDEPEND="sys-libs/zlib"
 DEPEND="${RDEPEND}
 	app-doc/doxygen"
 
-KEYWORDS="~amd64 ~x86"
+KEYWORDS="~amd64 ~ppc64 ~x86"
```

Comment:
you have a ppc64 machine at home? sweet!
jokes aside, you can't keyword ppc64 unless you're on the ppc64 AT